models/patientmodel.js

```diff
@@ -15,6 +15,7 @@ const patientSchema = new Schema({
     ref: "Doctor"
   }
 });
+//changes are made

```

Comment:
what a great changes you have made
keep it up very up

Reply:
Thank you